### PR TITLE
removed obsolete bootstrap classes

### PR DESCRIPTION
**Pull request by Vaibhav-afk**
Pull Request for Issue #31218

### Summary of Changes

- I did changes in:

      1. default_login.php.
      2. renderfield.php

### Output:
Capture

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -97,7 +97,7 @@
     "recursive-readdir": "^2.2.2",
     "rimraf": "^3.0.2",
     "rollup": "^2.42.1",
-    "rollup-plugin-vue": "^5.1.9",
+    "rollup-plugin-vue": "^6.0.0",
```
Comment (dgrammatiko) [inline package.json]: Please undo this, you're breaking the MEdia manager App here!!!

**Review by dgrammatiko — CHANGES_REQUESTED** [review]

**Comment by drmenzelit** [comment]
Did you checked how the login page look after your changes? You are changing more classes than necessary for the issue mentioned.
And do your changes have side effects on other forms?

**Comment by Vaibhav-afk** [comment]
I did the changes please review.

**Comment by ceford** [comment]
The original issue mentioned Bootstrap 4 but Joomla 4 is now using Bootstrap 5 and form-group has been removed. So this PR is wrong!<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32865">issues.joomla.org/tracker/joomla-cms/32865</a>.</sub>

**Comment by chmst** [comment]
https://github.com/joomla/joomla-cms/issues/31218#issuecomment-726579628  

Thank you for contributing. You cannot know this, of course, but as you see in the comment above it is not as easy task. There are 3rd party extension which use these classes for their own js and css - so we cannot remove classes as you did in the default_login.

**Comment by ceford** [comment]
@chmst I think I see: control-group was changed to form-group when Joomla 4 planned to use Bootstrap 4 and the recent decision to use Bootstrap 5 has come too late to change all 170 instances of form-group to mb-3 in Joomla core extensions. I don't see where third party extensions come in. If they have to change control-group anyway why not change to mb-3. I guess there are more complications than I can shake a stick at.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32865">issues.joomla.org/tracker/joomla-cms/32865</a>.</sub>

**Comment by brianteeman** [comment]
This probably should be closed

**Comment by Quy** [comment]
Closing per https://github.com/joomla/joomla-cms/pull/32865#issuecomment-808940864, Thank you for your contribution.

**Pull request closed without merging**